content/en/coec/index.adoc

```diff
@@ -0,0 +1,53 @@
+---
+title: "Code of Ethics and Conduct"
+description: "ACL Special Interest Group on Turkic Languages Code of Ethics and Conduct"
+draft: false
+toc: true
+images: []
+contributors: []
+---
+
+:toc:
+
+== Introduction
+
+SIGTURK is an ACL Special Interest Group on Turkic Languages that aims to foster community brainstorming and collaboration on events and resources for the public good in open source. As these processes involve sync and async communication, this document includes professional standards all participants of SIGTURK and SIGTURK-related dialogue must uphold to promote formal reliability of SIGTURK outputs, respect, competence, responsibility, and integrity while recognizing the rapidly changing society and the necessity to exercise more considerable caution in this spirit by every participant.
+
+SIGTURK also recognizes the following documents as authoritative, and their reading should interpret the authoring organization's name as SIGTURK under all respect to the license of the individual page:
```

Comment:
What does this mean? 

> their reading should interpret the authoring organization's name as SIGTURK under all respect to the license of the individual page

Reply:
@j0ma this is a common practice: these documents use their orgs' name as the address of ethics but we want readers to read those with SIGTURK instead

Reply:
OK that makes sense! Could we rephrase a little bit to make it clearer? Maybe 

> SIGTURK also recognizes the following list of documents as authoritative and they should be read and interpreted with SIGTURK as the authoring organization. 

Not really sure how to rephrase this bit "under all respect to the license of the individual page".

Happy to keep it as is too, though. I just thought it was a bit hard to understand at first but your explanation does make sense.

Reply:
Good point, I'll take some time to reflect on that